### PR TITLE
add deleteMessageSeconds to helers.banMember

### LINKED ISSUE #2408: [api-docs] Document ban `delete_message_seconds` and remove `reason` (#5219)
A new commit was made into the api-docs repo: https://github.com/discord/discord-api-docs/commit/7e3e3a3de46a909c9a5bc04404f0df9a806e7271
No details given.

This is a bot created issue.

### PR DESCRIPTION
**Pull request by lts20050703**
closes #2408 
upstream https://github.com/discord/discord-api-docs/commit/7e3e3a3de46a909c9a5bc04404f0df9a806e7271

### CODE REVIEW

**Comment by Skillz4Killz** [comment]
Add reason back, it was removed support from a json prop but we never used it as a json prop. Our system would remove the reason json prop and make it into a header

**Comment by Skillz4Killz** [comment]
Add unit test for this new property

**Comment by lts20050703** [comment]
@Skillz4Killz please re-review

File: helpers/members/banMember.ts [hunk 1]
```diff
@@ -6,19 +6,20 @@ export async function banMember(bot: Bot, guildId: bigint, id: bigint, options?:
...
+      delete_message_seconds: options?.deleteMessageSeconds,
+      reason: options?.reason,
+    },
   );
 }
 
 /** https://discord.com/developers/docs/resources/guild#create-guild-ban */
 export interface CreateGuildBan {
-  /** Number of days to delete messages for (0-7) */
+  /** Number of days to delete messages for (0-7) (deprecated) */
```
Comment (Skillz4Killz) [inline helpers/members/banMember.ts:19]: If this is really deprecated just remove it, in dd we don't keep things that are going to be killed anyway

**Review by Skillz4Killz — DISMISSED** [review]

**Review by Skillz4Killz — APPROVED** [review]

**Pull request merged**